app/lending/components/Icons/CryptoIcons.tsx

```diff
@@ -0,0 +1,225 @@
+    viewBox="0 0 32 32"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <circle cx="16" cy="16" r="16" fill="#6F788B" />
+    <path
+      d="M16 5.99985L15.7337 6.90569V20.0989L16 20.3652L22.4646 16.5591L16 5.99985Z"
+      fill="#C1CCF3"
+    />
+    <path
+      d="M16 5.99985L9.53543 16.5591L16 20.3652V13.7437V5.99985Z"
+      fill="white"
+    />
+    <path
+      d="M16 21.5879L15.8512 21.769V26.7314L16 26.9999L22.4702 17.7838L16 21.5879Z"
+      fill="#C1CCF3"
+    />
+    <path d="M16 26.9999V21.5879L9.53543 17.7838L16 26.9999Z" fill="white" />
+    <path d="M16 20.3652L22.4646 16.5591L16 13.7437V20.3652Z" fill="#8197EE" />
+    <path
+      d="M9.53543 16.5591L16 20.3652V13.7437L9.53543 16.5591Z"
+      fill="#C1CCF3"
+    />
+  </svg>
+)
+
+export const USDTIcon: FC = () => (
+  <svg
+    width="32"
+    height="32"
+    viewBox="0 0 32 32"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <circle cx="16" cy="16" r="16" fill="#26A17B" />
+    <path
+      d="M17.922 17.383V17.38C17.85 17.387 17.576 17.406 17.096 17.406C16.701 17.406 16.418 17.39 16.311 17.383V17.386C13.843 17.281 12.021 16.896 12.021 16.437C12.021 15.977 13.843 15.592 16.311 15.484V17.032C16.423 17.046 16.716 17.069 17.104 17.069C17.595 17.069 17.85 17.039 17.922 17.032V15.484C20.387 15.592 22.204 15.977 22.204 16.437C22.204 16.896 20.387 17.281 17.922 17.383"
+      fill="white"
+    />
+    <path
+      d="M17.922 15.258V13.949H21.565V10.5H10.687V13.949H14.331V15.258C11.501 15.391 9.5 16.003 9.5 16.727C9.5 17.452 11.501 18.062 14.331 18.195V24.5H17.922V18.195C20.747 18.062 22.75 17.452 22.75 16.727C22.75 16.003 20.747 15.391 17.922 15.258"
+      fill="white"
+    />
+  </svg>
+)
+
+export const DAIIcon: FC = () => (
+  <svg
+    width="32"
+    height="32"
+    viewBox="0 0 32 32"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <circle cx="16" cy="16" r="16" fill="#F5AC37" />
+    <path d="M9.55553 15.9999H7V17.7777H9.55553V15.9999Z" fill="white" />
+    <path d="M25 15.9999H22.4445V17.7777H25V15.9999Z" fill="white" />
+    <path d="M15.9999 7V9.55553H17.7777V7H15.9999Z" fill="white" />
+    <path d="M15.9999 22.4445V25H17.7777V22.4445H15.9999Z" fill="white" />
+    <path
+      d="M16 10.6665H11.9989C10.1567 10.6665 8.66669 12.1565 8.66669 13.9987V18.0009C8.66669 19.8431 10.1567 21.3331 11.9989 21.3331H20.0011C21.8433 21.3331 23.3334 19.8431 23.3334 18.0009V13.9987C23.3334 12.1565 21.8433 10.6665 20.0011 10.6665H16ZM16 10.6665V21.3331M11.0333 15.9999H20.9777"
+      fill="#FFFDFA"
+    />
+    <path
+      d="M16 10.6665H11.9989C10.1567 10.6665 8.66669 12.1565 8.66669 13.9987V18.0009C8.66669 19.8431 10.1567 21.3331 11.9989 21.3331H20.0011C21.8433 21.3331 23.3334 19.8431 23.3334 18.0009V13.9987C23.3334 12.1565 21.8433 10.6665 20.0011 10.6665H16ZM16 10.6665V21.3331M11.0333 15.9999H20.9777"
+      stroke="white"
+      strokeWidth="1.25"
+    />
+  </svg>
+)
+
+export const iconsBySymbol: Record<string, FC> = {
```

Comment:
We already have such icons. If you want to move them in one file, maybe then need remove old one?